src/ui/digital-exchange/components/list/ComponentList.js

```diff
@@ -36,6 +95,7 @@ class ComponentList extends Component {
 ComponentList.propTypes = {
   onWillMount: PropTypes.func,
   loading: PropTypes.bool,
+  listViewMode: PropTypes.string,
```

Comment:
IMO better naming it `viewMode`, because (if I understood) it's the view mode of the component list (and so it could be either `list-view` or `grid-view`)